modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SelectionControls.js

```diff
@@ -161,14 +162,32 @@ const SelectionControls = ({
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
+	const selectedItemsLabel = `${sub(
+		Liferay.Language.get('x-of-x'),
+		selectedItems,
+		itemsTotal
+	)} ${itemType} ${Liferay.Language.get('selected')}`;
```

Comment:
Is this right from a language POV? I mean, it will alway be `x-of-y itemType Selected` in any language? or the order may differ in some of them (like `x-of-y selected itemType`)? If soy we should create a new lang property for it.

cc @marcoscv-work

Reply:
Thanks for taking this into account. I agree, so I change it!